### PR TITLE
Code action test should wait for diagnostics

### PR DESCRIPTION
**Pull request by jessienguyen2312**
Closes https://github.com/microsoft/vscode-flake8/pull/223#issuecomment-1731992484
Added time out procedure to def test_command_code_action(code, contents, command) based on the example on test_linting.py

### CODE REVIEW

**Comment by karthiknadig** [comment]
@jessienguyen2312 Thank you for the contribution. You have to do two more things:
1. Can you follow this comment and accept the CLA? https://github.com/microsoft/vscode-flake8/pull/223#issuecomment-1731992484
2. You may need to run `black ./src/test/python_tests` and `isort --profile black ./src/test/python_tests`

**Comment by jessienguyen2312** [comment]
@microsoft-github-policy-service agree

**Review by karthiknadig — APPROVED** [review]

File: src/test/python_tests/test_code_actions.py [hunk 1]
```diff
@@ -14,6 +15,7 @@
 TEST_FILE_PATH = constants.TEST_DATA / "sample1" / "sample.py"
 TEST_FILE_URI = utils.as_uri(str(TEST_FILE_PATH))
 LINTER = utils.get_server_info_defaults()["name"]
+TIMEOUT = 10  # 10 seconds
```
Comment (eleanorjboyd) [inline src/test/python_tests/test_code_actions.py:18]: is this in seconds or milliseconds?

Comment (karthiknadig) [reply src/test/python_tests/test_code_actions.py:18]: It is seconds. Python in most cases uses seconds and floating point for milliseconds

**Review by eleanorjboyd — APPROVED** [review]

**Pull request merged**